src/clj/ataru/applications/application_store.clj

```diff
@@ -4,7 +4,8 @@
             [camel-snake-kebab.extras :refer [transform-keys]]
             [schema.core :as s]
             [oph.soresu.common.db :as db]
-            [yesql.core :refer [defqueries]]))
+            [yesql.core :refer [defqueries]]
+            [clojure.java.jdbc :as jdbc :refer [with-db-transaction]]))
```

Comment:
Either use alias for each member of the namespace or then refer everything. So either :as or :refer but not both.